redaxo/src/addons/users/pages/roles.php

```diff
@@ -21,7 +21,7 @@
 if ($func == '') {
     $title = rex_i18n::msg('user_role_caption');
 
-    $list = rex_list::factory('SELECT id, name FROM ' . rex::getTablePrefix() . 'user_role');
+    $list = rex_list::factory('SELECT id, name FROM ' . rex::getTablePrefix() . 'user_role ORDER BY name', 100);
```

Comment:
warum das limit am ende?

Reply:
Ich finde das Paginieren in den Listen ungünstig und habe mich hier gleich an die Modulliste gehalten.
https://github.com/redaxo/redaxo/blob/master/redaxo/src/addons/structure/plugins/content/pages/modules.modules.php#L385